rllib/policy/dynamic_tf_policy_v2.py

```diff
@@ -1047,3 +1048,41 @@ def gradients(self, optimizer, loss):
                 return self.compute_gradients_fn(optimizers[0], losses[0])
         else:
             return super().gradients(optimizers, losses)
+
+    @override(TFPolicy)
+    @OverrideToImplementCustomLogic_CallToSuperRecommended
+    def get_state(self):
+        # Legacy Policy state (w/o keras model and w/o PolicySpec).
+        state = super().get_state()
+        # Add this Policy's spec so it can be retreived w/o access to the original
+        # code.
+        state["policy_spec"] = PolicySpec(
```

Comment:
New:
- Add full PolicySpec to state, such that a policy can be recovered w/o(!) knowing the config/class.
- Add native (keras/torch) model files resulting from a `save` operation on this native model.